.travis.yml

```diff
@@ -12,5 +12,17 @@ php:
   - 7.1
   - hhvm
 
+matrix:
+  include:
+    - php: 7
+      env: PHPSTAN=1
+    - php: 7.1
+      env: PHPSTAN=1
```

Comment:
No need to add a second PHP 7 job running both PHPUnit and PHPSTAN IMO. It increases the matrix for nothing.